### DISCUSSION TITLE
How do I get "View in your space" to work?

### DISCUSSION THREAD

**Discussion by DOEHOONLEE** [google/model-viewer#4243] [Q&A]
![IMG_5913](https://user-images.githubusercontent.com/93510050/233890892-cfb86e0a-87ec-4e40-a302-cade8deefeea.png)

When I tap "View in your space" on my phone, nothing happens. The last time I tried in 2022, it worked fine, but it does nothing now. What is happening?

I'm on using iPhone 13 on iOS version 13.

**Comment by elalish** [comment]
I don't know, but that's a 4-year-old OS - have you tried updating?

**Reply by DOEHOONLEE** [reply]
Oh! I'm sorry. That was a typo! I meant to say version 16.3 :(

**Reply by DOEHOONLEE** [reply]
<img width="1037" alt="Screenshot 2023-05-14 at 3 52 30 PM" src="https://github.com/google/model-viewer/assets/20305442/cbe89509-7370-4c59-a464-930f55cb750a">
<img width="436" alt="Screenshot 2023-05-14 at 3 52 41 PM" src="https://github.com/google/model-viewer/assets/20305442/ba10e97c-dace-4253-86fe-001d728d8003">

I updated from 16.3 to 16.4 just now.
I connected my iPhone to Macbook to see the developer tool and I see no error :(
I am not sure what is going on.

**Reply by elalish** [reply]
Are you using our auto-generation or your own `ios-src`? Sometimes Apple has breaking changes to USDZ where old files stop working and you need to re-export them.
